src/test/resources/testResult/newRowsInserted/discrete_ground_water.csv

```diff
@@ -1,25 +1,25 @@
-"9BDA141822744BA5E0530100007FD075","393215104490002",null,null,null,null,null,null,null,null,"Elevation, GW, NGVD29|ft","62610","GW level, steel tape","S","774.72",ft,"0.01",null,null,Unspecified,HIF-242820,"2019-06-20 16:36:00",Source: Reporting Agency,true,null,"2a9e559615e04ca7b7191d5ca99e7332",false,"[\"Static\"]","900",Working,null,NGVD29
-"9BDA141822744BA5E0530100007FD075","393215104490002",null,null,null,null,null,null,null,null,"Water level, depth LSD|ft","72019","GW level, steel tape","S","20.28",ft,"0.01",null,null,Unspecified,HIF-242820,"2019-06-20 16:36:00",Source: Reporting Agency,true,null,"2a9e559615e04ca7b7191d5ca99e7332",false,"[\"Static\"]","900",Working,null,Local Assumed Datum
-"9BDA141822744BA5E0530100007FD075","393215104490002",null,null,null,null,null,null,null,null,"Water level, depth MP|ft","61055","GW level, steel tape","S","120.28",ft,"0.01",null,null,Unspecified,HIF-242820,"2019-06-20 16:36:00",Source: Reporting Agency,true,null,"2a9e559615e04ca7b7191d5ca99e7332",false,"[\"Static\"]","900",Working,null,depth MP
-"f251791c-4c7f-4a7c-9480-997f2eeb0b94","393215104490002",null,null,null,null,null,null,null,null,"Elevation, GW, NAVD88|ft","62611","GW level, steel tape","S","771.41",ft,"0.01",null,null,Unspecified,HIF-242820,"2019-11-13 18:32:00",Source: Reporting Agency,true,null,"2a9e559615e04ca7b7191d5ca99e7332",false,"[\"Static\"]","900",Working,null,NAVD88
-"f251791c-4c7f-4a7c-9480-997f2eeb0b94","393215104490002",null,null,null,null,null,null,null,null,"Elevation, GW, NGVD29|ft","62610","GW level, steel tape","S","774.48",ft,"0.01",null,null,Unspecified,HIF-242820,"2019-11-13 18:32:00",Source: Reporting Agency,true,null,"2a9e559615e04ca7b7191d5ca99e7332",false,"[\"Static\"]","900",Working,null,NGVD29
-"f251791c-4c7f-4a7c-9480-997f2eeb0b94","393215104490002",null,null,null,null,null,null,null,null,"Water level, depth LSD|ft","72019","GW level, steel tape","S","20.52",ft,"0.01",null,null,Unspecified,HIF-242820,"2019-11-13 18:32:00",Source: Reporting Agency,true,null,"2a9e559615e04ca7b7191d5ca99e7332",false,"[\"Static\"]","900",Working,null,Local Assumed Datum
-"f251791c-4c7f-4a7c-9480-997f2eeb0b94","393215104490002",null,null,null,null,null,null,null,null,"Water level, depth MP|ft","61055","GW level, steel tape","S","120.52",ft,"0.01",null,null,Unspecified,HIF-242820,"2019-11-13 18:32:00",Source: Reporting Agency,true,null,"2a9e559615e04ca7b7191d5ca99e7332",false,"[\"Static\"]","900",Working,null,depth MP
+"46686b86-77c8-4fef-8d72-a994a6a267a5","393215104490001",null,null,null,null,null,null,null,null,"Elevation, GW, NAVD88|ft","62611","GW level, steel tape","S","5777.77",ft,"0.01","ReferencePrimary",null,Unspecified,HIF-242820,"2020-03-05 18:42:00",Source: Reporting Agency,true,null,"2a9e559615e04ca7b7191d5ca99e7332",false,"[\"Static\"]","900",Working,null,NAVD88
```

Comment:
Looks like the data does not have any  read_type other than "ReferencePrimary" This is probably good for a first
pass. No need to hold up testing. Hopefully at some point, we will get an end to end test, and eliminate mocking up
data like this.